### PR TITLE
Use command line flags in sql bank example.

### PR DESCRIPTION
**Pull request by thschroeter**
Similar to flag usage in the kv example.

### CODE REVIEW

File: examples/sql_bank/main.go [hunk 1]
```diff
@@ -32,23 +33,21 @@ import (
...
 
-const (
-	maxTransfer = 999
-	numAccounts = 999
-	concurrency = 5
-	aggregate   = true
-	usePostgres = false
-)
+var maxTransfer = flag.Int("maxTransfer", 999, "Maximum amount to transfer in one transaction.")
+var numAccounts = flag.Int("numAccounts", 999, "Number of accounts.")
```
Comment (bdarnell) [inline examples/sql_bank/main.go]: The flags should be named with hyphens, as they are in kv_bank: `var numAccounts = flag.Int("num-accounts",...)`.

**Comment by bdarnell** [comment]
LGTM

File: examples/sql_bank/main.go (hunk 1 above)
Comment (thschroeter) [inline examples/sql_bank/main.go]: Oh, I missed that. Corrected now.

**Comment by tamird** [comment]
Thank you!

**Pull request merged**